### PR TITLE
update comments for LTI config in env.json (iss: #1223)

### LINKED ISSUE #1223: update comments for LTI config
For Canvas-hosted instance the URLs below can be domain-specific or generic. Canvas supports either. It's better to keep it generic to support vanity URL changes. 

See: https://canvas.instructure.com/doc/api/file.lti_dev_key_config.html

```json
"auth_login_url": "https://canvas.test.instructure.com/api/lti/authorize_redirect",
"auth_token_url": "https://canvas.test.instructure.com/login/oauth2/token",
"key_set_url": "https://canvas.test.instructure.com/api/lti/security/jwks",
```

This issue is to update the comments in `env_sample.json`.

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #1223

### CODE REVIEW

**Comment by lsloan** [comment]
So, this PR is updates to the comments.  We need an issue to update the documentation in the wiki, too.  I'll create that.

File: config/env_sample.json [hunk 1]
```diff
@@ -93,19 +93,19 @@
...
     "/*  LTI 1.3 configuration":"*/",
-    "/*  The first key of LTI_CONFIG is your Canvas domain name":"*/",
+    "/*  The first key of LTI_CONFIG is your environment specific Canvas url and not domain specific":"*/",
     "LTI_CONFIG": {
         "https://canvas.instructure.com": [
             {
                 "/*The default set of variables for LTI validation":"*/",
                 "default": true,
                 "/*LTI Dev Key from Canvas":"*/",
                 "client_id": "17700000000000111",
```
Comment (lsloan) [inline config/env_sample.json:103]: The comments below are now too specific and confusing.  Not everyone who may set up MyLA knows the difference between domain-specific and environment-specific.

We should simply give examples of the hosts allowed in the URLs.

File: config/env_sample.json [hunk 2]
```diff
@@ -93,19 +93,19 @@
     "/*  LTI is disabled by default": "*/",
     "STUDENT_DASHBOARD_LTI": false,
     "/*  LTI 1.3 configuration":"*/",
-    "/*  The first key of LTI_CONFIG is your Canvas domain name":"*/",
+    "/*  The first key of LTI_CONFIG is your environment specific Canvas url and not domain specific":"*/",
```
Comment (lsloan) [inline config/env_sample.json:96]: Is this so?  If the `canvas*.instructure.com` URLs are used for launch, then to MyLA it appears to have be running there?  It doesn't redirect to a domain-specific URL?

File: config/env_sample.json [hunk 3]
```diff
@@ -93,19 +93,19 @@
...
     "LTI_CONFIG": {
         "https://canvas.instructure.com": [
             {
                 "/*The default set of variables for LTI validation":"*/",
                 "default": true,
                 "/*LTI Dev Key from Canvas":"*/",
                 "client_id": "17700000000000111",
-                "/*fill this out for your institution.":"*/",
+                "/*fill this out environment specific and need not to be domain specific.":"*/",
```
Comment (lsloan) [inline config/env_sample.json:104]: We need only this single, specific comment for the three URLs that follow it.

```suggestion
                "/*Allowed hosts for the following 3 URLs: canvas.instructure.com, canvas.beta.instructure.com, canvas.test.instructure.com":"*/",
```

File: config/env_sample.json [hunk 4]
```diff
@@ -93,19 +93,19 @@
...
                 "auth_login_url": "https://canvas.instructure.com/api/lti/authorize_redirect",
-                "/*fill this out for your institution.":"*/",
+                "/*fill this out environment specific and need not to be domain specific.":"*/",
```
Comment (lsloan) [inline config/env_sample.json:106]: Remove this comment.

```suggestion
```

File: config/env_sample.json [hunk 5]
```diff
@@ -93,19 +93,19 @@
...
                 "auth_token_url": "https://canvas.instructure.com/login/oauth2/token",
-                "/*fill this out for your institution.":"*/",
+                "/*fill this out environment specific and need not to be domain specific.":"*/",
```
Comment (lsloan) [inline config/env_sample.json:108]: Remove this comment.

```suggestion
```

**Review by lsloan — CHANGES_REQUESTED** [review]

File: config/env_sample.json (hunk 2 above)
Comment (pushyamig) [reply config/env_sample.json]: The issuer id is always independent of the domain specific and canvas is strict on that. The LTI launch will fail if you use domain specific URL. I hope that answers your Question

**Review by pushyamig — COMMENTED** [review]
I think I resolved all the comment of yours

**Review by lsloan — APPROVED** [review]
I see that for the most part you used the suggestions I'd given earlier.  I think the comments for these items are pretty clear now.

**Pull request merged**